src/Pheanstalk.php

```diff
@@ -383,7 +383,7 @@ private function reconnect()
     {
         $this->connection->disconnect();
 
-        if ($this->using != PheanstalkInterface::DEFAULT_TUBE) {
+        if ($this->using !== null && $this->using != PheanstalkInterface::DEFAULT_TUBE) {
```

Comment:
This reverts your change and adds strict comparison.
```suggestion
        if ($this->using !== PheanstalkInterface::DEFAULT_TUBE) {
```